src/garrysmod/gamemodes/zombiesurvival/gamemode/zombieclasses/zombie.lua

```diff
@@ -283,4 +283,12 @@ end
 
 if CLIENT then
 	CLASS.Icon = "zombiesurvival/killicons/zombie"
+
+	function CLASS:PrePlayerDraw(pl)
+		render.SetColorModulation(0.5, 0.9, 0.5)
+	end
+
+	function CLASS:PostPlayerDraw(pl)
+		render.SetColorModulation(1, 1, 1)
+	end
```

Comment:
Interesting, this doesn't appear to carry over to the ragdolls upon player death, and was missing from the hands of the player model.

Not sure if it's worth looking into now since we're copying the same functionality from the Agile Dead class.